blessings/tests.py

```diff
@@ -117,7 +115,7 @@ def test_zero_location():
     t = TestTerminal(stream=StringIO(), force_styling=True)
     with t.location(0, 0):
         pass
-    eq_(t.stream.getvalue(), unicode_cap('sc') +
+    assert t.stream.getvalue() == unicode_cap('sc' +
```

Comment:
The original line contained `unicode_cap('sc') +` whilst your change brings `unicode_cap('sc' +`. I wonder if this is the root cause of the test fail.